activiti-cloud-services-audit/activiti-cloud-services-audit-mongo/src/test/java/org/activiti/cloud/services/audit/mongo/rest/TestMvcClient.java

```diff
@@ -0,0 +1,205 @@
+package org.activiti.cloud.services.audit.mongo.rest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.util.List;
+
+import org.springframework.hateoas.Link;
+import org.springframework.hateoas.LinkDiscoverer;
+import org.springframework.hateoas.LinkDiscoverers;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.ResultMatcher;
+
+public class TestMvcClient {
```

Comment:
I think you can move the query [TestMvcClient](https://github.com/Activiti/activiti-cloud-query-service/blob/develop/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/test/java/org/activiti/cloud/services/query/rest/support/TestMvcClient.java) to the module [activiti-cloud-services-test](https://github.com/Activiti/activiti-cloud-service-common/tree/develop/activiti-cloud-services-test) in order to get it shared between query, audit and audit mongo. Then add a dependency with `test` scope to this module.

Reply:
Thank you for your review comment.
I fixed.